frontend/src/features/admin-form/settings/mutations.ts

```diff
@@ -95,7 +96,9 @@ export const useMutateFormSettings = () => {
         // Show toast on success.
         const isNowPublic = newData.status === FormStatus.Public
         const toastStatusMessage = isNowPublic
-          ? `Your form is now open.\n\nIf you expect a large number of responses,  [AutoArchive your mailbox](https://go.gov.sg/form-prevent-bounce) to avoid losing any of them.`
+          ? newData.responseMode === FormResponseMode.Encrypt
+            ? `Your form is now open.\n\nStore your secret key in a safe place. If you lose your secret key, all your responses will be lost permanently.`
+            : `Your form is now open.\n\nIf you expect a large number of responses,  [AutoArchive your mailbox](https://go.gov.sg/form-prevent-bounce) to avoid losing any of them.`
```

Comment:
nit: should we separate out this computation for readability?

Reply:
ok done